m2install.sh

```diff
@@ -1629,20 +1627,18 @@ function updateWebsiteIndexFile()
 {
   local websiteCode="$1"
   local websiteDir="websites/${websiteCode}";
-  local codeLine='$params[\\Magento\\Store\\Model\\StoreManager::PARAM_RUN_CODE] = '"'${websiteCode}';";
-  local typeLine='\$params[\\Magento\\Store\\Model\\StoreManager::PARAM_RUN_TYPE] = '"'website';";
-  sed -i "28 i ${codeLine}" "${websiteDir}/index.php"
-  sed -i "29 i ${typeLine}" "${websiteDir}/index.php"
-  sed -i "s/[\/][.][.][\/]/\/..\/..\//" "${websiteDir}/index.php"
+  local codeLine='$_SERVER[\\Magento\\Store\\Model\\StoreManager::PARAM_RUN_CODE] = '"'${websiteCode}';";
+  local typeLine='$_SERVER[\\Magento\\Store\\Model\\StoreManager::PARAM_RUN_TYPE] = '"'website';";
+  sed -i "36 i ${codeLine}" "${websiteDir}/index.php"
+  sed -i "37 i ${typeLine}" "${websiteDir}/index.php"
+  sed -i "s/[\/]app[\/]bootstrap[.]php/\/..\/..\/app\/bootstrap.php/" "${websiteDir}/index.php"
 }
 
 function createFileStructure()
 {
   local websiteDir="$1";
   [ ! -d "${websiteDir}" ] && mkdir "${websiteDir}";
-  [ ! -f websites/index.php.tmp ] \
-    && curl -s -o "websites/index.php.tmp" https://raw.githubusercontent.com/magento/magento2/2.2/pub/index.php
```

Comment:
What if index.php is modified on dumps?
Will it fail updating storecode and type